openstack/arc/charts/mosquitto/templates/deployment.yaml

```diff
@@ -31,10 +31,22 @@ spec:
       containers:
       - name: {{ .Chart.Name }}
         image: "{{required ".Values.global.registry is missing" .Values.global.registry }}/{{ .Values.image.repository }}:{{.Values.image.tag }}"
-        command: [mosquitto, -c, /mosquitto/mosquitto.conf]
+        command:
+          - sh
+          - -ec
+          - |
+            cp /mosquitto_tls/tls.crt /tmp/mosquitto_tls_crt.livenessprobe
+            exec mosquitto -c /mosquitto/mosquitto.conf
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         ports:
         - containerPort: {{ .Values.service.internalPort }}
+        livenessProbe:
+          exec:
+            command:
+            - cmp
+            - -s
+            - /mosquitto_tls/tls.crt
+            - /tmp/mosquitto_tls_crt.livenessprobe
```

Comment:
I would add the some timeout/interval changes as from the example:
```
initialDelaySeconds: 60
periodSeconds: 60
timeoutSeconds: 3
```

We don't need to execute the check that often and the increased timeout (default: 1s) avoids false positive due to cpu/I/o contention.

Reply:
agreed and commited